feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -270,15 +270,28 @@ func (tc *testCase) verifyInstall(ctx context.Context, t *testing.T) {
 	}
 
 	if tc.dualSup {
-		if got, want := r.GetVerifyStandby().GetVerifyResponse().GetActivationFailMessage(), ""; got != want {
-			t.Errorf("OS.Verify Standby ActivationFailMessage: got %q, want %q", got, want)
-		}
+		var counter int
+		const maxRetries = 15
+		for i := 1; i < maxRetries; i++ {
```

Comment:
maybe consider moving the `tc.osc.Verify` at the end of the for loop scope. Because you already ran it once above, and you already have some result in the `r` for comparison. Only if it does not match or produces nothing, then you retry it at the end, right?